### ISSUE TITLE
Unable to add new tools to config

### ISSUE THREAD

**Issue by Strakh97** [RiskyKen/Armourers-Workshop#106]
So, first off _absolutely love_ the mod. Like, as a player that cares more about aesthetic than pretty much anything? This is perfect.
Second, I have encountered an issue. See, I notice you can add tools etc to the sword list? Yeah, for whatever reason it won't let me add the Draconic Staff of Power (DraconicEvolution:draconicDistructionStaff) or the Mekanism Atomic Disassembler (Mekanism:AtomicDisassembler), which is a shame because I tend to use multitools, and am at the endgame stage where I'd rather not switch back to individual tools. I don't know if I'm doing anything wrong, all I am doing is adding the tool ID in the same format as is already written.. Anybody else had this issue, or am I doing it wrong, or is there a fix I've missed completely?

**Comment by RiskyKen** [comment]
When the game is loading do you get any errors in the log like "Invalid sword override in config file" or "Unable to override item renderer for" you may need to turn on showing the log on whatever launcher you are using.

**Comment by Strakh97** [comment]
Ok, in regards to the two items, I get the following:
[Armourer's Workshop]: Overriding render on - Mekanism:AtomicDisassembler
[Armourer's Workshop]: Storing custom item renderer for: AtomicDisassembler
And
[Armourer's Workshop]: Overriding render on - DraconicEvolution:draconicDistructionStaff
[Armourer's Workshop]: Storing custom item renderer for: draconicDistructionStaff
But nothing that seems to indicate any kind of error at all..

I also get the same two lines for the following items:
TwilightForest:item.iceSword
TwilightForest:item.fierySword
TwilightForest:item.glassSword
TConstruct:cutlass
TConstruct:rapier
TConstruct:battleaxe
TConstruct:cleaver
TConstruct:broadsword
Botania:elementiumSword
_(Note: I added none of these as extra. They were added in automatically)_

And then there are a load of other weapons added. Just for "_In case you'd need it_" here's all the log says in reference to Armourers Workshop: http://pastebin.com/H9XktWky
(And the standard load path for the skin library on world load)

Upon testing the re-skinning for the Ice Sword, a TiCo Cutlass and the Elementuim sword, I could re-skin them as expected.
